Editor/Module Management/ModuleInstaller.cs

```diff
@@ -135,5 +137,27 @@ private static void AppendScriptingSymbol()
             var newDefineString = string.Join(";", symbols.ToArray());
             PlayerSettings.SetScriptingDefineSymbolsForGroup(target, newDefineString);
         }
+        
+        private static void ValidateModules()
+        {
+            var packageList = GetPackageList();
+            var allModuleInstalled = true;
+            foreach (var module in ModuleList.Modules)
+            {
+                if (packageList.All(x => x.name != module.name))
+                {
+                    allModuleInstalled = false;
+                }
+            }
+           
+            if (allModuleInstalled)
+            {
+                SDKLogger.Log(TAG, "All the modules are installed successfully. Ready Player Me avatar system is ready to use.");
```

Comment:
These hard-coded log messages could be moved to a const. 
minor thing though :D